cli/cmd/lql.go

```diff
@@ -50,47 +50,72 @@ var (
 
 	// queryCmd represents the lql parent command
 	queryCmd = &cobra.Command{
-		Aliases: []string{"lql"},
 		Use:     "query",
+		Aliases: []string{"lql", "queries"},
 		Short:   "run and manage queries",
-		Long: `Run and manage queries.
+		Long: `Run and manage Lacework Query Language (LQL) queries.
 
-NOTE: This feature is not yet available!`,
+To provide customizable specification of datasets, Lacework provides the Lacework
+Query Language (LQL). LQL is a human-readable text syntax for specifying selection,
+filtering, and manipulation of data.
+
+Currently, Lacework has introduced LQL for configuration of AWS CloudTrail policies
+and queries. This means you can use LQL to customize AWS CloudTrail policies only.
+For all other policies, use the previous existing methods.
+
+Lacework ships a set of default LQL queries that are available in your account.
+
+For more information about LQL, visit:
+
+   https://support.lacework.com/hc/en-us/articles/4402301824403-LQL-Overview
+
+To view all LQL queries in your Lacework account.
+
+    lacework lql ls
```

Comment:
Did we want to standardize on `lacework query` here or was `lacework lql` intentional to illustrate the `lql` alias?

Reply:
it was intentional but better not to do it, I'll put `query` instead